### PR TITLE
update recipe to 0.9.21

### PR DESCRIPTION
**Pull request by newville**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by newville** [comment]
@conda-forge-admin, please rerender

**Comment by newville** [comment]
@conda-forge/core please merge, adding me as maintainer with write access.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+{% set sha256 = "ee14ba2211cda1c76114e3e7b552cdd57e940309203d5f4106e6d6f2c2346a2e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/4f/d5/ce490afa6f6ab1bb97ffd66a80bfb032d22407fc215d60b21ca174e25c51/asteval-0.9.21.tar.gz
```
Comment (CJ-Wright) [inline recipe/meta.yaml:11]: Please leave the jinja2 in if possible. Did the existing jinja2 not work?

Comment (newville) [reply recipe/meta.yaml]: @CJ-Wright  restored to better URL. I swear that when I tried that with `curl` it failed, but it did just work for me from a browser.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+{% set sha256 = "ee14ba2211cda1c76114e3e7b552cdd57e940309203d5f4106e6d6f2c2346a2e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}--{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipe/meta.yaml:11]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

**Pull request merged**